webui/react/src/components/DynamicIcon.tsx

```diff
@@ -39,17 +39,11 @@ const DynamicIcon: React.FC<Props> = ({ name, size = 70, style }: Props) => {
     return 10;
   }, [size]);
 
-  const borderRadius = useMemo(() => {
```

Comment:
What does this impact?

Reply:
The DynamicIcon is only used to represent Workspaces. This change makes the icon border radius always be 2px, which matches the border radius of the new cards, instead of being 4px or 8px depending on the size of the icon.

Reply:
So this touches all places we use this icon?

Reply:
Yes.